ush/yaml_finalize

```diff
@@ -214,10 +214,16 @@ with open(yfile2, 'r') as infile, open(yfile, 'w') as outfile:
             buffer_zone.append(line)
 
         if not in_buffer_zone:
-            if not do_envar_radar_ref and ", cloud_liquid_water, cloud_liquid_ice, rain_water, snow_water, graupel, equivalent_reflectivity_factor" in line:
+            if not do_radar_ref and ", cloud_liquid_water, cloud_liquid_ice, rain_water, snow_water, graupel, equivalent_reflectivity_factor" in line:
                 line = line.replace(", cloud_liquid_water, cloud_liquid_ice, rain_water, snow_water, graupel, equivalent_reflectivity_factor", "")
-            elif not do_envar_radar_ref and "    - equivalent_reflectivity_factor" in line:
-                line = line.replace("    - equivalent_reflectivity_factor\n", "")
+            if not do_radar_ref and ", upward_air_velocity" in line:
+                line = line.replace(", upward_air_velocity", "")
+            if not do_radar_ref and line.strip() == "- equivalent_reflectivity_factor" in line:
+                line = ""
```

Comment:
I think `line=""` will add an extra empty line as the original line contain `\n`.
Could you double check this?
_(although it is fine that we add an extra empty line)_

Reply:
ah, that's great to hear!